core/scheduler/scheduler.go

```diff
@@ -690,7 +690,18 @@ func (s *Scheduler) submitValidatorRegistrations(ctx context.Context) error {
 		})
 	}
 
-	return s.eth2Cl.SubmitValidatorRegistrations(ctx, registrations)
+	submitRegistrationCounter.Add(1)
+
+	err := s.eth2Cl.SubmitValidatorRegistrations(ctx, registrations)
+	if err != nil {
+		submitRegistrationErrors.Add(1)
+
+		log.Error(ctx, "Failed to submit validator registrations", err)
```

Comment:
BTW, something interesting to keep in mind and we have no influence on:
MEV-boost will report success on registrations to the BN (and hence to us) if at least 1 succeeds. This is weird behaviour IMO as it should result in failure if at least 1 fails.

Reply:
We should start talking to MEV directly...